Assets/Scenes/robysoftScene.unity

```diff
@@ -448,7 +448,7 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   nextScene: titleScene
   sceneDuration: 5
-  isSkippable: 0
+  isSkippable: 1
```

Comment:
Was this just a temporary change so you didn't have to keep watching it or did you intend to change this?

Reply:
I was doing it for testin' but I still think it should be skippable IMHO! Maybe it's an option we savce to disk, if they've watched the intro all the way through once we let them skip?

Reply:
I like the disk pref option as a compromise - although one potential complication is Devin might make a intro track that's timed to match this scene and the 'flash' of the intro video. We can always go ahead and just turn off the skipping later if we want though.